static/src/javascripts/projects/commercial/modules/lotame-data-extract.js

```diff
@@ -17,14 +20,10 @@ const shouldLoadLotame = (): boolean => {
         (edition === 'UK' || edition === 'INT')
     );
 };
+let initialised: boolean = false;
 
 // Fetches Lotame Data for the Ozone project
-// and stores in in window.OzoneLotameData
-const init = (): Promise<void> => {
-    if (!shouldLoadLotame || commercialFeatures.isSecureContact) {
-        return Promise.resolve();
-    }
-    return loadScript('//ad.crwdcntrl.net/5/c=13271/pe=y/var=OzoneLotameData')
+const loadLotamePromise = () => loadScript('//ad.crwdcntrl.net/5/c=13271/pe=y/var=OzoneLotameData')
```

Comment:
Should you enforce the type? I understand that the return value might sometimes be `window.LOTCC.bcp()` and other times `void`.

```suggestion
const loadLotamePromise = (): Promise => loadScript('//ad.crwdcntrl.net/5/c=13271/pe=y/var=OzoneLotameData')
```

Reply:
Good catch! Added `Promise<void>`